### DISCUSSION TITLE
mac电脑超级烫手，只能当做玩具，对电脑性能要求很高

### DISCUSSION THREAD

**Discussion by wiselinpm** [GitSquared/edex-ui#1082] [Questions]
做的很漂亮，就是爆性能，开小会儿温度飙升到88度，太恐怖

**Comment by qinyihao** [comment]
Translation into English:

You worked very well, but it takes up a lot in the system's performance. It's horrible that the temperature skyrocketed to 88 degrees Celsius in a short while.

**Reply by GitSquared** [reply]
Thanks for the translation. Performance varies a lot between hardware and software configurations, and while it has improved a lot over time, I can't do miracles: this is old software, on a stack that isn't made for performance, displaying many widgets fed by lots of info.
